doc/quick-install.md

```diff
@@ -1,64 +1,89 @@
 
-```sh
-apt-get update
-```
+A certificate is required to enable secure communication between conference participants and your service.
+
+Most conference participants will only be able to join your service if it uses a certificate generated by a trusted [certificate authority](https://en.wikipedia.org/wiki/Certificate_authority).
+
+Jitsi Meet will offer to generate a self-signed certificate for you during installation.  You can accept this offer if you do not have an existing certificate, but some participants may experience difficulty joining your service.
+
+Optionally you can generate a trusted certificate for free using the non-profit [Let's Encrypt](https://letsencrypt.org/) service using the following command:
 
-(If you get an error:
-E: The method driver /usr/lib/apt/methods/https could not be found.
-run:
 ```sh
-apt-get install apt-transport-https
+/usr/share/jitsi-meet/scripts/install-letsencrypt-cert.sh
 ```
-)
 
 ### Install Jitsi Meet
 
 Note: Something to consider before installation is how you're planning to serve Jitsi Meet. The installer will check if Nginx or Apache is present (with this order) and configure a virtualhost within the web server it finds to serve Jitsi Meet. If none of the above is found it then configures itself to be served via jetty. So if for example you are planning on deploying Jitsi Meet with a web server, you have to make sure to install the server **before** installing jitsi-meet.
 
 ```sh
+# Ensure support is available for apt repositories served via HTTPS
+apt-get install apt-transport-https
+
+# Retrieve the latest package versions across all repositories
+apt-get update
+
+# Perform jitsi-meet installation
 apt-get -y install jitsi-meet
 ```
 
-During the installation, you will be asked to enter the hostname of the Jitsi Meet instance. If you have a FQDN hostname for the instance already set up in DNS, enter it there. If you don't have a resolvable hostname, you can enter the IP address of the machine (if it is static or doesn't change).
+During the installation, you will be asked to enter the hostname of the Jitsi Meet instance.
 
-This hostname (or IP address) will be used for virtualhost configuration inside the Jitsi Meet and also, you and your correspondents will be using it to access the web conferences.
+If the system has a fully-qualified DNS name (such as `https://jitsi.example.org`), enter it when prompted.
 
-### Generate a Let's Encrypt certificate
+Alternatively, you can enter the IP address of the machine (if it is static or doesn't change).
 
-Simply run the following in your shell
+You'll need to share this hostname or IP address with conference participants who will use your service.
 
-```sh
-/usr/share/jitsi-meet/scripts/install-letsencrypt-cert.sh
-```
+### Confirm that your installation is working
+
+Launch a web browser (Chrome, Chromium or latest Opera) and enter the hostname or IP address from the previous step into the address bar.
```

Comment:
NB: From reading other issues, it sounds like Safari support including video may be enabled soon.  At that point it might be nice to update this to list the three highest browsers based on [market share](https://en.wikipedia.org/wiki/Usage_share_of_web_browsers#Summary_tables) which are (in order, at the time of writing): Chrome, Safari, Firefox

Reply:
Once that lands we need to update ... there is a notification to be removed, unsupported browser detection ... many things, but lets land it and we will be doing that.